scripts/generators/patternfly-4-component/templates/component/{{componentName}}.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
+import styles from '@patternfly/patternfly-next/{{typeDir}}/{{componentName}}/styles.css';
```

Comment:
This may not always be 1 to 1 although it is probably easier to delete or modify than not having it at all.

Reply:
I had same thoughts, rather to have something and remove it rather than to add it.